README.md

```diff
@@ -86,3 +334,24 @@ Apps below 10 libyears are considered to be healthy apps. We regularly rescue pr
 
 ## References
 J. Cox, E. Bouwers, M. van Eekelen and J. Visser, Measuring Dependency Freshness in Software Systems. In Proceedings of the 37th International Conference on Software Engineering (ICSE 2015), May 2015 https://ericbouwers.github.io/papers/icse15.pdf
+
+
+## Contributing
+
+If you want to contribute to this project, please fork it, create a dedicated branch to make your changed and simply raise a pull request that targets this repository.
+
+## License
```

Comment:
I think we might need to keep the MIT license from the original repo: https://github.com/nasirhjafri/libyear/blob/master/setup.py#L26

Reply:
Indeed, I have put MIT text grabbed from https://opensource.org/license/mit